src/warnet/k8s.py

```diff
@@ -359,3 +388,43 @@ def write_file_to_container(pod_name, container_name, dst_path, data):
         return True
     except Exception as e:
         print(f"Failed to copy data to {pod_name}({container_name}):{dst_path}:\n{e}")
+
+
+def get_kubeconfig_value(jsonpath):
+    command = f"kubectl config view --minify -o jsonpath={jsonpath}"
+    return run_command(command)
+
+
+def get_namespaces() -> list[V1Namespace]:
+    sclient = get_static_client()
+    try:
+        return [
+            ns
+            for ns in sclient.list_namespace().items
+            if ns.metadata.name not in KUBE_INTERNAL_NAMESPACES
+        ]
+
+    except ApiException as e:
+        if e.status == 403:
+            ns = sclient.read_namespace(name=get_default_namespace())
+            return [ns]
+        else:
+            return []
+
+
+def get_namespaces_by_prefix(prefix: str) -> list[V1Namespace]:
```

Comment:
if the plan is to eventually move to labels, we could call this function `get_namespaces_by_type` or something similar, and then have this function be a black box. Right now its using prefix, but we could refactor just this function in the future to actually use labels/annotations.